js/app/modules/layout/articleStack.js

```diff
@@ -239,8 +239,10 @@ const ArticleStack = new Module.Class({
 
     _on_history_changed: function () {
         let item = HistoryStore.get_default().get_current_item();
-        if (item.page_type !== Pages.ARTICLE)
+        if (item.page_type !== Pages.ARTICLE) {
+            this.get_children().forEach((view) => view.set_active(false));
```

Comment:
well doing that now won't work because `set_active` plays the video and I don't want the video to start playing when the view loads - only when the user actually clicks play